keyboards/kbdfans/kbd67/rev2/rules.mk

```
@@ -30,4 +30,4 @@ UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 
-LAYOUTS = 65_ansi 65_iso 65_ansi_blocker 65_ansi_blocker_split_bs 65_ansi_split_bs
+LAYOUTS = 65_ansi 65_iso 65_ansi_blocker 65_ansi_blocker_split_bs 65_ansi_split_bs 65_ansi_split_bs_split_space
```

```suggestion
LAYOUTS = 65_ansi 65_iso 65_ansi_blocker 65_ansi_blocker_split_bs 65_ansi_split_bs
```
I don't see this under `layouts/default/`.

---

I added this layout as it fits what I soldered on my board, do I need to make additional changes?

---

What would be the correct way to achieve what I am trying to do, then? Add KC_TRANS for each key that I did not solder to and use the LAYOUT macro with each key?

---

You don't have to do anything except the suggestion above; these layouts don't exist under `layouts/` so they should not be specified here.

---

Oh, but I can keep the definition in `keyboards/kbdfans/kbd67/rev2/rev2.h` ?

---

Yes. `LAYOUTS` is not meant to list every possible layout a board supports, only the ones that are common enough to have an entry in `layouts/`. This feature allows you to have a keymap you can flash onto any board with the same layout, without needing to copy it into each board's `keymaps/` folder.

---

Thanks for the explanation, I get it now :)